presto-main/src/main/java/com/facebook/presto/security/AllowAllSystemAccessControl.java

```diff
@@ -61,6 +64,12 @@ public void checkQueryIntegrity(Identity identity, AccessControlContext context,
     {
     }
 
+    @Override
+    public AuthorizedIdentity selectAuthorizedIdentity(Identity identity, AccessControlContext context, String userName, List<X509Certificate> certificates)
```

Comment:
This is because "TestAllowAllSystemAccessControl" requires all methods from SystemAccessControl need to be overriden; so the dummy identity (original identity) is returned.